src/components/application_manager/src/commands/mobile/system_request.cc

```diff
@@ -559,7 +559,7 @@ void SystemRequest::Run() {
       return;
     }
 
-    CFormatterJsonBase::jsonValueToObj(root, sm_object);
+    CFormatterJsonBase::jsonValueToObj(root, sm_object, "");
```

Comment:
@jacobkeeler may be it would be better to use default argument? Or overloaded function? Mandatory explicit setting key it confusing.